src/main/java/com/customerApi/api/controllers/CustomerUpdateController.java

```diff
@@ -0,0 +1,39 @@
+package com.customerApi.api.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import org.springframework.web.bind.annotation.RestController;
+
+import com.customerApi.api.modals.Customer;
+import com.customerApi.api.services.UpdateCustomerService;
+
+
+
+
+@RestController
+public class CustomerUpdateController {
+	
+	private static final Logger logger = LoggerFactory.getLogger(CustomerUpdateController.class);
+	
+	@Autowired
+	UpdateCustomerService service;
+	
+	@PutMapping("/updateCustomer")
+	
+	public ResponseEntity <Customer> customerUpdate(@RequestBody Customer customer)
+	{
+		logger.info("Redirect to update service :"+customer);
```

Comment:
The proper way of printing log is below, its an example
_For example, if you want to print 1 value"_
logger.info("Customer Name {}, sign-in in", customer.getName());

_For example, if you want to print 2 values or multiple"_
logger.info("Customer First Name {}, last name {}, sign-in in", customer.getFirstName(), customer.getLastName());

Reply:
Thank you for your feedbacks. I have done the changes as per your comments